resources/run_adf.sh

```diff
@@ -280,6 +297,43 @@ else
   fi
 fi
 
+manage_compute_node() {
+  if [ $ARG = $START ]; then
+    check_compute_node $MONGO_DOWNLOAD_DIR
+    echo "Starting a compute $MONGOD"
+    mkdir -p ./artifacts/compute-mode-mongod-data
+
+    # Start compute mongod
+    if [[ $OS =~ ^CYGWIN ]]; then
+      # mongod does not have --fork option on Windows, using nohup
+      nohup $MONGO_DOWNLOAD_DIR/bin/mongod -f ./testdata/config/mongod.conf --logpath $(cygpath -m $LOGS_PATH/compute_mongod.log) &
+      echo $! > $TMP_DIR/compute_node.pid
+    else
+      $MONGO_DOWNLOAD_DIR/bin/mongod -f ./testdata/config/mongod.conf --logpath $LOGS_PATH/compute_mongod.log  --pidfilepath $TMP_DIR/compute_node.pid --fork
+    fi
+
+    waitCounter=0
+    while : ; do
+        check_compute_node
+        if [[ $? -eq 0 ]]; then
+            break
+        fi
+        if [[ "$waitCounter" -gt $TIMEOUT ]]; then
+            echo "ERROR: Local mongod did not start under $TIMEOUT seconds"
```

Comment:
```suggestion
            echo "ERROR: Local compute node mongod did not start under $TIMEOUT seconds"
```
Differentiating which mongod is failing